ygot/testdata/exschemapath/exschemapath.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package exschemapath;
+
+option go_package = "github.com/openconfig/ygot/ygot/testdata/exschemapath";
+
+import "github.com/openconfig/ygot/proto/ywrapper/ywrapper.proto";
+import "github.com/openconfig/ygot/proto/yext/yext.proto";
+
+message Root {
+    System system = 1; 
+    message InterfaceKey {
+        string name = 1 [(yext.schemapath) = "/interfaces/interface/config/name|/interfaces/interface/name"];
```

Comment:
This case will be covered later? Should we have a placeholder test like the other unsupported cases?

Reply:
I've added support for this kind of message -- since it seemed trivial to do so.